index.html

```diff
@@ -0,0 +1,401 @@
+            remote work.
+            <br />
+            I've also been a good helping hand to a lot of developers in
+            different communities!
+            <br />
+            if you'd like to get in contact about potential job opportunities,
+            please reach out!
+          </p>
+          <a
+            href="https://docs.google.com/document/d/1lYq0Kb50TujzJzDWGobIchRI17JMnAiZMF9OBQgNU68/edit?usp=sharing"
+            class="link"
+            target="_blank"
+            rel="noreferrer noopener"
+          >
+            Get my resume
+          </a>
+          <img class="about-me-icon" src="./media/svg/icon-2.svg" alt="Icon" />
+        </section>
+        <hr />
+        <div class="about-me-section stacks">
+          <ul>
+            <li class="languages list-title">
+              Languages
+              <ul>
+                <li>JavaScript</li>
+                <li>TypeScript</li>
+                <li>Ruby</li>
+                <li>HTML</li>
+                <li>CSS</li>
+                <li>Sass</li>
+              </ul>
+            </li>
+          </ul>
+          <ul>
+            <li class="frameworks list-title">
+              Frameworks
+              <ul>
+                <li>React & Redux</li>
+                <li>Material-UI</li>
+                <li>Jest</li>
+                <li>Bootstrap</li>
+                <li>Ruby on Rails</li>
+                <li>RSPec</li>
+              </ul>
+            </li>
+          </ul>
+          <ul>
+            <li class="skills list-title">
+              Skills
+              <ul>
+                <li>GitHub</li>
+                <li>CodePen</li>
+                <li>GitLab</li>
+                <li>Terminal</li>
+                <li>WSL</li>
+                <li>Remote Pair-Programming</li>
+                <li>Teamwork</li>
+                <li>Mentoring</li>
+                <li>Code Reviews</li>
+                <li>Mobile/Responsive Development</li>
+              </ul>
+            </li>
+          </ul>
+        </div>
+      </div>
+      <section class="contact" id="contact">
+        <h2 class="intro">
+          I'm always interested in hearing about new projects, so if you'd like
+          to chat please get in touch.
+        </h2>
+        <form id="form" action="https://formspree.io/f/mrgjqpap" method="POST">
+          <label hidden>Name</label>
+          <input
+            class="form-fields"
+            type="text"
+            name="name"
+            id="name"
+            maxlength="30"
+            placeholder="Full Name"
+            required
+          />
+          <label hidden>Email Address</label>
+          <input
+            class="form-fields"
+            type="email"
+            name="Email Address"
+            id="email-address"
+            placeholder="Email Address"
+            required
+          />
+          <small id="email-invalidation-message"></small>
+          <label hidden>Subject</label>
```

Comment:
Great job following the Figma templates 👍 , however, there's a slight variation between your design and that of the original one given. 

- [ ] The space between your form and the footer needs to be fixed. I've attached a screenshot to show you the difference. I suggest you make the changes, to complement the good job you've done so far 👏
- [ ] The Hyperlink color of your media links is slightly different from the figma design.  I've attached a screenshot to show you the difference. I suggest you make the changes so you  complement the good job you've done so far 👏
## Template design:
![F111](https://user-images.githubusercontent.com/50990925/211005423-e89d4be5-a434-4d0b-982b-a59e781544da.png)

## Your work So far:
![figm222](https://user-images.githubusercontent.com/50990925/211006383-9758ea53-8ef4-47f8-9d98-a38583ae1224.png)